samples/nextjs/src/pages/[[...path]].SSR.tsx

```diff
@@ -23,6 +30,19 @@ const SitecorePage = ({ notFound, layoutData, componentProps }: SitecorePageProp
     return <NotFound />;
   }
 
+  useEffect(() => {
+    // Do not trigger client tracking when pages are requested by Sitecore XP instance:
+    // - no need to track in Edit and Preview modes
+    // - in Explore mode all requests will be tracked by Sitecore XP out of the box
+    if (isPreview) return;
+
+    if (tracked) return;
```

Comment:
This line probably deserves a comment. Also, could be combined with previous check e.g. `if (isPreview || tracked) return;`

Reply:
Done